telemetry_core/src/main/java/com/newrelic/telemetry/SenderConfiguration.java

```diff
@@ -67,36 +69,77 @@ public static SenderConfigurationBuilder builder(String defaultUrl, String baseP
     private boolean auditLoggingEnabled = false;
     private String secondaryUserAgent;
 
+    /** Create a new Builder, with the default URL and base path. */
```

Comment:
This javadoc (also line 60) adds no value, only further maintenance in the future.

Reply:
Please discuss in #156